packages/indexer-client/src/types/clientTypes.ts

```diff
@@ -682,13 +685,18 @@ export interface GetIndexerLeaderboardResponse {
 }
 
 export interface GetIndexerLeaderboardParticipantParams {
-  contestId: number;
+  contestIds: number[];
   subaccount: Subaccount;
 }
 
+// contestId -> IndexerLeaderboardParticipant;
+export type IndexerTieredLeaderboardParticipant = Record<
```

Comment:
Hm, we don't necessarily need to be using this for tiered leaderboards (we can use it across unrelated contests, right?), I'm not sure we need this type, as we could define the `Record` directly inline.

Related question - if a subaccount is not eligible, would the key be omitted from the response? If so, this might be good to shout-out in the JSDoc comment

Reply:
yup, agree -- will make the changes